src/dashboard/dashboard.py

```diff
@@ -0,0 +1,84 @@
+"""dashboard.py - Streamlit app for displaying the stats dashboard.
+"""
+
+from log_keeper.get_config import get_config
+import pymongo
+import streamlit as st
+import pandas as pd
+import matplotlib.pyplot as plt
+
+st.markdown("# 661 VGS Dashboard")
+st.sidebar.markdown("# 661 VGS Dashboard")
+
```

Comment:
Then up here we can add the subfunction (make sure there's two lines either side):

```suggestion
def format_data_for_display(raw_df):
    """Format the data for display in Streamlit.

    Args:
        data_df (pd.DataFrame): The data to be formatted.

    Returns:
        pd.DataFrame: The formatted data.
    """
    # Format the date and time columns.
    data_df = raw_df.copy()
    data_df["TakeOffTime"] = raw_df["TakeOffTime"].dt.strftime("%H:%M")
    data_df["LandingTime"] = raw_df["LandingTime"].dt.strftime("%H:%M")

    # Format the date column.
    data_df["Date"] = raw_df["Date"].dt.strftime("%Y-%m-%d")

    # Remove _id column.
    data_df = data_df.drop(columns=["_id"])
    return data_df
```